micro_sam/__init__.py

```diff
@@ -5,4 +5,22 @@
 .. include:: ../doc/python_library.md
 """
 
-from .__version__ import __version__
+__version__ = "0.1.2.post1"
+
+from .sample_data import (
+    sample_data_image_series,
+    sample_data_wholeslide,
+    sample_data_livecell,
+    sample_data_hela_2d,
+    sample_data_3d,
+    sample_data_tracking,
+)
+
+__all__ = (
```

Comment:
Is it necessary to use the `__all__` here for the napari plugin? This might shadow other things in `pdoc` (I haven't checked yet, so not sure if it's actually a problem.)

Reply:
No, I don't think it is necessary at all. We can get rid of it if you like!

Reply:
Update: it is fine, I can get rid of this :)